hr_expense_advance_overdue_reminder/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Employee Advance Overdue Reminder",
+    "version": "15.0.1.0.0",
+    "category": "Human Resources",
+    "author": "Ecosoft, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "summary": "Simple mail overdue employee advance reminder",
+    "website": "https://github.com/OCA/hr-expense",
+    "depends": [
+        "hr_expense_advance_clearing",
+        "hr_expense_advance_clearing_sequence",
```

Comment:
```suggestion
        "hr_expense_advance_clearing_sequence",
```
I think you can depend only `hr_expense_advance_clearing_sequence` because it already depends `hr_expense_advance clearing`.